modules/installation-azure-config-yaml.adoc

```diff
@@ -108,25 +114,34 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 //storage type as `io1` and set `iops` to `2000`.
 <6> Specify a list of zones to deploy your machines to. For high availability, specify at least two zones.
 <7> Specify the name of the resource group that contains the DNS zone for your base domain.
-ifdef::vnet[]
+ifdef::vnet,private[]
 <8> If you use an existing VNet, specify the name of the resource group that contains it.
 <9> If you use an existing VNet, specify its name.
 <10> If you use an existing VNet, specify the name of the subnet to host the control plane machines.
 <11> If you use an existing VNet, specify the name of the subnet to host the compute machines.
+endif::vnet,private[]
+ifdef::private[]
+<12> You can customize your own outbound routing. Configuring user-defined routing prevents exposing external endpoints in your cluster. User-defined routing for egress requires deploying your cluster to an existing VNet.
```

Comment:
Similar to Abhinav's review, there is no external endpoints when using a private cluster. This could be explained once in the document, but definitely needs to be there somewhere.

I would replace the second sentence with something like "Configuring user-defined routing will leverage any type of outbound connectivity applied to your pre-existing VNET or subnets such as routing or NAT. When enabled, it will not create the objects used by the default outboundType (public IP and load balancer).